hbase-client/src/main/java/org/apache/hadoop/hbase/client/Admin.java

```diff
@@ -2254,4 +2254,28 @@ boolean snapshotCleanupSwitch(final boolean on, final boolean synchronous)
    */
   boolean isSnapshotCleanupEnabled() throws IOException;
 
+
+  /**
+   * Retrieve online slowlog responses from in memory ringbuffer maintained by
+   * RegionServers
+   *
+   * @param serverNames Server names to get slowlog responses from
+   * @return online slowlog response list
+   * @throws IOException if a remote or network exception occurs
+   */
+  List<SlowLogPayload> getSlowLogResponses(final List<ServerName> serverNames)
+      throws IOException;
+
+  /**
+   * Clears online slowlog response buffer maintained by RegionServers for all
+   * serverNames provided in the argument
+   *
+   * @param serverNames List of Server names to clean slowlog responses from
+   * @return List of booleans representing if onlinse slowlog response buffer is cleaned
+   *   from each RegionServer
+   * @throws IOException if a remote or network exception occurs
+   */
+  List<Boolean> clearSlowLogsResponses(final List<ServerName> serverNames)
```

Comment:
The naming rules of clearSlowLogsResponses() and getSlowLogResponses() should be the consistent, the character "s" after "clearSlowLog"  should be removed.